networking/p2p/src/integration-test/java/tech/pegasys/artemis/networking/p2p/jvmlibp2p/BeaconBlocksByRangeIntegrationTest.java

```diff
@@ -62,63 +60,51 @@ public void shouldSendEmptyResponsePreGenesisEvent() throws Exception {
 
   @Test
   public void shouldSendEmptyResponseWhenNoBlocksAreAvailable() throws Exception {
-    initChainStorageClient(storageClient1);
+    beaconChainUtil.initializeStorage();
     final List<BeaconBlock> response = requestBlocks(storageClient1.getBestBlockRoot());
     assertThat(response).isEmpty();
   }
 
   @Test
   public void shouldSendEmptyResponseWhenHeadBlockRootDoesNotMatchAnyBlock() throws Exception {
-    initChainStorageClient(storageClient1);
+    beaconChainUtil.initializeStorage();
+    beaconChainUtil.createAndImportBlockAtSlot(1);
     final List<BeaconBlock> response = requestBlocks(Bytes32.ZERO);
     assertThat(response).isEmpty();
   }
 
   @Test
   public void shouldSendEmptyResponseWhenHeadBlockRootIsNotOnCanonicalChain() throws Exception {
-    initChainStorageClient(storageClient1);
-    final BeaconBlock nonCanonicalBlock = addBlock(1);
-    final List<BeaconBlock> response = requestBlocks(nonCanonicalBlock.hash_tree_root());
+    beaconChainUtil.initializeStorage();
+    final BeaconBlock nonCanonicalBlock = beaconChainUtil.createAndImportBlockAtSlot(1).getBlock();
```

Comment:
I'd expect `createAndImportBlockAtSlot` to advance the chain.  Maybe the util could automatically update the chain head, and you could explicitly reset it here with something like:  `storageClient1.updateBestBlock(genesisRoot, genesisSlot)`.